src/app/test/business/dad_jokes_service_test.dart

```diff
@@ -0,0 +1,125 @@
+import 'dart:convert';
+
+import 'package:app/access/dad_jokes/dad_jokes_repository.dart';
+import 'package:app/access/dad_jokes/data/dad_joke_content_data.dart';
+import 'package:app/access/dad_jokes/favorite_dad_jokes_repository.dart';
+import 'package:app/business/dad_jokes/dad_joke.dart';
+import 'package:app/business/dad_jokes/dad_jokes_service.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+import '../mocks/dad_jokes_data_mock.dart';
```

Comment:
Can we import those as `package` to stay consistent or not?

Reply:
not sure what you mean by that

Reply:
Example import 'package:shared_preferences/shared_preferences.dart';

Reply:
We can usually use the same way with our own files